docs/modules/ROOT/pages/object-storage/usage.adoc

```diff
@@ -0,0 +1,49 @@
+= How to Use an AppCat Object Bucket
```

Comment:
I suggest we use K8up as an example here. Just the definition of a one-time `Backup` object referencing the secret parameters. What do you think?

Reply:
I've added a k8up backup as an example. I feel it's not quite optimal though...

What do you think?

Reply:
Precisely! This is perfect. Thanks!